docs/integration_resources/software/nuke.md

```diff
@@ -13,10 +13,68 @@ from shift.tools import nshift
 nshift.show()
 ```
 
+## Shift installation in Nuke
+
+In order to have Shift installed inside Nuke and be able to open the Shift UI with a menu entry, is required to setup Shift in the user init and menu python files. It can be added in the user init and menu files from the nuke preference folder ([home directory]/.nuke) or a custom init and menu files inside your pipeline. If the directory were this files are stored is a path read by Nuke, they will be automatically executed on startup.

```

Comment:
I don't fully get the second sentence:
" It can be added in the user init and menu files from the nuke preference folder ([home directory]/.nuke) or a custom init and menu files inside your pipeline."

With "It" you mean "Shift"? And with "added" you mean "imported"? And pipeline you mean in a different directory with a personal folder structure?

I left a suggestion, but feel free to rephrase it if needed:
```suggestion
To install Shift and open the Shift UI with a menu entry within Nuke, it is required to set up Shift in the user init and menu Python files from Nuke. Shift can be added to the user init and menu files from Nuke's preference folder (*<home directory>/.nuke*) or to a custom init and menu files inside a custom folder structure. If the directory where these files are stored is a path read by Nuke, they will be automatically executed on startup.

```

Reply:
Nuke automatically execute the init.py and menu.py that it found in the folders added to the Nuke_Path if I remember correctly.

The TDs usually have a main init and menu files where they configure the whole startup. Yes, with it I mean Shift, we can replace it. So what I mean with added is like another tool to be added/imported to Nuke. 

I will commit your suggestion